custom-connectors/Actsoft/README.md

```diff
@@ -0,0 +1,92 @@
+# Actsoft Mobile Workforce Plus
```

Comment:
@rdameron Please move your connector to `certified-connectors` folder if you want it to be get certified. We do not certify the connectors goes to `custom-connectors` folder.

Reply:
Moved file to the correct folder. I'll meet with the dev team on Monday to resolve the other items.

Reply:
Done